### DISCUSSION TITLE
Integration with RITA?

### DISCUSSION THREAD

**Discussion by xNeck** [cisagov/Malcolm#340] [General Discussions]
Is there any chance we will see RITA scanner on Malcolm? Ty 😃

**Comment by mmguero** [comment]
Earlier this year I would have said "there's no way", but I just did some reading on the [what's new](https://www.activecountermeasures.com/free-tools/rita/) of RITA and apparently they've ditched the MongoDB requirement which was the major non-starter for me. I don't know if we will or won't, but I've [created an issue](https://github.com/idaholab/Malcolm/issues/587) for us to investigate it in the future.

**Comment by mmguero** [comment]
Tracking the enhancement at idaholab/Malcolm#587, closing this discussion in lieu of that issue.
